src/Sulu/Bundle/RouteBundle/Command/UpdateRouteCommand.php

```diff
@@ -32,6 +32,7 @@ protected function configure()
     {
         $this->setName('sulu:route:update')
             ->addArgument('entity', InputArgument::REQUIRED)
+            ->addArgument('locale', InputArgument::REQUIRED)
```

Comment:
the problem is that at this place we have no idea which locales should be present here.